packages/sit-onyx/src/components/OnyxCheckbox/OnyxCheckbox.vue

```diff
@@ -26,9 +30,12 @@ const isChecked = computed({
       <input
         v-model="isChecked"
         class="onyx-checkbox__input"
+        :class="{ 'onyx-checkbox__input--touched': isTouched }"
```

Comment:
fyi it's not uncommon with bem to move the modifier on the block level -> line 28
```html
  <label class="onyx-checkbox"
    :class="{ 'onyx-checkbox--touched': isTouched }"
  >
```

this might make you drop a few of the ":has" selectors


(just a suggestion, if it makes things more complicated, leave it as-is 🙂)

Reply:
I had this at first but imho it made things more complicated :D

Reply:
ok :)